forms-flow-web/src/components/Form/Item/Submission/Success/SuccessPage.js

```diff
@@ -646,6 +646,24 @@ export default React.memo(() => {
           </div>
         </>
       );
+    } else if (
+      search.includes(successTypes.DELETE_TELEWORK)
+    ) {
+      let name = search.replace("?type=DELETE_TELEWORK_", "");
```

Comment:
maybe use `successTypes.DELETE_TELEWORK` here, instead of the string. what do you think?

Reply:
that's good catch, Updated it. Please review.